### PR TITLE
update(boilerplate): boilerplate section

### LINKED ISSUE #220: Agregar información sobre Full Stack Boilerplate
Crear una pagina nueva que describa la razón y los beneficios del una plantilla para iniciar un proyecto nuevo. 

enfoquémonos en el  repositorio full stack  pero incluyamos enlaces al  repo front end y backend . 

Fusionemos esta pagina nueva con los documentos pre-existentes 

https://guias.eoscostarica.io/docs/plantilla-para-aplicacion-web

https://guias.eoscostarica.io/docs/plantilla-para-backend

Related to : #205

### PR DESCRIPTION
**Pull request by Edgar097**
### GH Issue

Resolves #220  

### Steps to test
1. Go to boilerplate
1. Check content

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

**Comment by xavier506** [comment]
@Edgar097  Please have a look at some initial content I prepared in english , I think we can model this page off that and then add content where we think its missing

https://github.com/eoscostarica/guide.eoscostarica.io/issues/205

**Comment by Edgar097** [comment]
Let me know if the title of the page should be different, we can add links to the React Js , Docker and GraphQL tutorials but maybe later when all of them are merge.

File: docs/boilerplate.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
+id: boilerplate
+title: Boilerplate de Proyecto
+sidebar_label: Boilerplate de Proyecto
+---
+## ¿Qué es un Boilerplate?
+
+Un boilerplate es un marco o plantilla para el desarrollo de un proyecto. Es una estructura de proyecto ya definida de antemano que sirve para agilizar la organización y el desarrollo del software. Acelera la integración de soluciones y formatos que sirven para conducir el desarrollo de un nuevo proyecto a la producción.
+
+Hay tres repositorios de boilerplate: **FrontEnd**, **BackEnd** y **Full Stack**.
```
Comment (xavier506) [inline docs/boilerplate.md:10]: En EOS Costa Rica hemos creado tres repositorios boilerplate :

**Comment by xavier506** [comment]
> Let me know if the title of the page should be different, we can add links to the React Js , Docker and GraphQL tutorials but maybe later when all of them are merge.

@Edgar097  I like the idea of linking content!

File: docs/boilerplate.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+hapi.js (comúnmente conocido como hapi) son las siglas de HTTP API. Es un marco rico para crear aplicaciones y servicios. Fue diseñado originalmente para el rápido desarrollo de servicios de API RESTful usando JavaScript, pero desde entonces ha crecido hasta convertirse en un marco de aplicación web completo con características listas para usar plantillas, validación de entrada, autenticación, almacenamiento en caché y, más recientemente, soporte. para aplicaciones en tiempo real con soporte web socket.
+
+Obtenga más información en [https://hapi.dev/](https://hapi.dev/)
+
+### EOSIO
+
+EOSIO es un blockchain de tercera generación que permitirá alcanzar el objetivo de proporcionar un registro inmmutable, permanente, trazable y verificable de toda la actividad en los depósitos de contenedores. Estos datos serán públicos y podrán ser consumidos por cualquier tercero externo.
+
+### PostgresDB
+
```
Comment (xavier506) [inline docs/boilerplate.md:72]: Lets add some information on PostGres DB

File: docs/boilerplate.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+Hay tres repositorios de boilerplate: **FrontEnd**, **BackEnd** y **Full Stack**.
+
+## Front End
+
+[**EOS Costa Rica Frontend Boilerplate**](https://github.com/eoscostarica/webapp-boilerplate)
+
+"Front-end" se refiere a la interfaz de usuario y en el contexto de una aplicación web es la aplicación que se ejecuta en el navegador web de los usuarios.
+
+Utilizamos [React JS](https://reactjs.org/), una biblioteca de JavaScript para la construcción de interfaces de usuario de alto rendimiento y basada en componentes que facilita el desarrollo de interfaces de usuario complejas con código predecible y fácil de depurar.
```
Comment (xavier506) [inline docs/boilerplate.md:18]: I think its better if this link points to the react tutorial page

File: docs/boilerplate.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+### Colecciones de Componentes React
+
+Hemos desarrollado una colección de componentes en React para EOSIO. Puede ver los componentes que hemos construido en nuestro [EOS Costa Rica React Components Storybook](https://guias.eoscostarica.io/docs/tutoriales/tutorial-react#componentes-en-eos-costa-rica)
+Para más información sobre los componentes puede visitar [Repositorio de Componentes](https://github.com/eoscostarica/eoscr-components)
+
+## BackEnd
+
+[**EOS Costa Rica Backend Boilerplate**](https://github.com/eoscostarica/backend-boilerplate)
+
+"Back-end" se refiere a la aplicación del servidor , se puede describir es la capa de acceso a los datos de una aplicación que contiene la lógica de negocio para la gestión de datos dentro de la aplicación.
```
Comment (leisterfrancisco) [inline docs/boilerplate.md:36]: 👾 Take a look to 👾
> del servidor , se puede

**Review by leisterfrancisco — DISMISSED** [review]

File: docs/boilerplate.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+title: Boilerplate de Proyecto
+sidebar_label: Boilerplate de Proyecto
+---
+## ¿Qué es un Boilerplate?
+
+Un boilerplate es un marco o plantilla para el desarrollo de un proyecto. Es una estructura de proyecto ya definida de antemano que sirve para agilizar la organización y el desarrollo del software. Acelera la integración de soluciones y formatos que sirven para conducir el desarrollo de un nuevo proyecto a la producción.
+
+En EOS Costa Rica hemos creado tres repositorios boilerplate: **FrontEnd**, **BackEnd** y **Full Stack**.
+
+## Front End
```
Comment (xavier506) [inline docs/boilerplate.md:12]: ## Front-End Boilerplate

File: docs/boilerplate.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+- [Apollo Client](https://www.apollographql.com/docs/react/v3.0-beta/)
+- [Material UI](https://material-ui.com/)
+- [i18 next](https://react.i18next.com/)
+
+### Colecciones de Componentes React
+
+Hemos desarrollado una colección de componentes en React para EOSIO. Puede ver los componentes que hemos construido en nuestro [EOS Costa Rica React Components Storybook](https://guias.eoscostarica.io/docs/tutoriales/tutorial-react#componentes-en-eos-costa-rica)
+Para más información sobre los componentes puede visitar [Repositorio de Componentes](https://github.com/eoscostarica/eoscr-components)
+
+## BackEnd
```
Comment (xavier506) [inline docs/boilerplate.md:32]: ## Back-End Boilerplate

File: docs/boilerplate.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+Obtenga más información en [Componentes EOSIO](https://guias.eoscostarica.io/docs/aprender-eosio/componentes-eosio)
+
+### PostgresDB
+
+*"PostgreSQL es un potente sistema de base de datos relacional de objetos de código abierto con más de 30 años de desarrollo activo que le ha ganado una sólida reputación por su fiabilidad, solidez y rendimiento".*
+
+PostgreSQL viene con muchas características destinadas a ayudar a los desarrolladores a crear aplicaciones, a los administradores a proteger la integridad de los datos y a crear entornos tolerantes a fallas, y a administrar sus datos sin importar cuán grande o pequeño sea el conjunto de datos.
+Obtenga más información en [PostgresDB](https://guias.eoscostarica.io/docs/herramientas-para-desarrolladores#postgresdb)
+
+## Full-Stack
```
Comment (xavier506) [inline docs/boilerplate.md:80]: ## Full-Stack Boilerplate

**Comment by xavier506** [comment]
@Edgar097  we should tray and standardize the way we write `Front End` , `Back End` , and `Full Stack`  what do you suggest?

**Comment by Edgar097** [comment]
> @Edgar097 we should tray and standardize the way we write `Front End` , `Back End` , and `Full Stack` what do you suggest?
Front-End is fine for me

**Review by xavier506 — APPROVED** [review]
Nice Job!

**Review by xavier506 — APPROVED** [review]
LGTM

File: docs/boilerplate.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+
+"Back-end" se refiere a la aplicación del servidor, se puede describir es la capa de acceso a los datos de una aplicación que contiene la lógica de negocio para la gestión de datos dentro de la aplicación.
+
+### Back-End Tech Stack
+
+- [Hasura](https://hasura.io/)
+- [Hapi](https://hapi.dev/)
+- [IPFS](https://ipfs.io/)
+- [PostgreSQL](https://www.postgresql.org/)
+- [EOS](https://eos.io/)
```
Comment (xavier506) [inline docs/boilerplate.md:44]: EOSIO

**Review by xavier506 — APPROVED** [review]

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by adriexnet — APPROVED** [review]

**Pull request merged**